installation/macos.md

```diff
@@ -1,11 +1,11 @@
 ---
```

Comment:
Renaming this file would require a change to:
https://github.com/openhab/website/blob/master/.vuepress/config.js#L209
as well.
This menu structure (at least for the docs & addons) should really be migrated to this repo at some point.

Reply:
Thanks for the comment @ghys! I've now created https://github.com/openhab/website/pull/105 as well.
I was unsure if it would pick this up automatically but apparently it does not. :wink: